--- a/src/components/Map.tsx
+++ b/src/components/Map.tsx
@@ -165,18 +165,18 @@
 interface MapProps {
   potholes: Pothole[];
   onMarkerClick: (pothole: Pothole) => void;
+  initialCenter: LatLngExpression;
 }

-const Map = ({ potholes, onMarkerClick }: MapProps) => {
-  const thaneMajiwada: LatLngExpression = [19.2076, 72.9645];
+const Map = ({ potholes, onMarkerClick, initialCenter }: MapProps) => {
   const initialZoom = 14;

   return (
     <MapContainer
-      center={thaneMajiwada}
+      center={initialCenter}
       zoom={initialZoom}
       zoomControl={false}
       style={{ height: '100vh', width: '100%' }}
@@ -186,7 +186,7 @@
         attribution='&copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors'
       />
       <Markers potholes={potholes} onMarkerClick={onMarkerClick} />
-      <CustomControls center={thaneMajiwada} zoom={initialZoom} />
+      <CustomControls center={initialCenter} zoom={initialZoom} />
     </MapContainer>
   );
 };
